.github/workflows/auto-update.yml

```
@@ -0,0 +1,148 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Need full history for version comparison
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+          cache: 'npm'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)